serialization/jsonld.md

```diff
@@ -44,29 +44,37 @@ discussion).
 The SPDX organization provides a global JSON-LD @context file that MUST be used
 universally for all SPDX JSON-LD files of a given SPDX version.
 
-The context is available under https://spdx.org/rdf/3.0.0/spdx-context.jsonld
+The context is available under <https://spdx.org/rdf/3.0.1/spdx-context.jsonld>
```

Comment:
Last I checked, we are not changing this URL for micro version?

Reply:
From the [2024-07-16 minutes](https://github.com/spdx/meetings/blob/main/tech/2024/2024-07-16.md), it looks like the decision was made to include the micro version?

I wasn't on the call myself, so I can't confirm the decision.

Reply:
I'm sure that the decision is to use micro/patch version (and it is already applied in #800). But I'm not sure about this specific URL for context file.

Maybe the reason we like to keep the URL of context file 3.0.0 because we don't want to create too much burden for users to updating the URL in the future?

- If we change the context file URL every micro/patch version, delivering updated content in the new context file to user will be more difficult/slower for each version -- as the user has to specifically changing the URL.

- If we keep "3.0.0" name for any 3.0.x version, we'll basically use "3.0.0" here the way "3.0" supposed to be. The actual content from this "3.0.0" URL will be the latest 3.0.x.

At the same time, it's quite weird to have the URL that way - because the versioning is not consistent with our IDs (which is now https://spdx.org/rdf/3.0.1/terms/Core, etc. and can be https://spdx.org/rdf/3.0.2/terms/Core, etc. in the future ..)

I know that this is a breaking one, but does it too late to have just `https://spdx.org/rdf/3.0/spdx-context.jsonld` instead ?
- Burden to update context file URL still remain for the next update (from `https://spdx.org/rdf/3.0.0/spdx-context.jsonld` to `https://spdx.org/rdf/3.0/spdx-context.jsonld`), but that will be the last time?
- We can still keep using IDs 3.0.x if we want; things may be less challenging if we can update the content of context file regularly

Reply:
2024-08-06 Tech call decided to go with `3.0.1`  https://github.com/spdx/meetings/pull/757